requirements-readthedocs.txt

```diff
@@ -6,21 +6,20 @@ Sphinx==1.2.3
 amqp==1.4.6
 anyjson==0.3.3
 billiard==3.3.0.19
-celery==3.1.17
-datastream==0.4.9
+celery==3.1.18
+datastream==0.4.10
 defusedxml==0.4.1
-django-celery==3.1.16
 django-classy-tags==0.6.1
 git+https://github.com/wlanslovenija/ultrajson@a8f0f0f1010956b27bf0c2cb5e52d85bb84e273a#egg=ujson-wlanslovenija
 git+https://github.com/wlanslovenija/django-tastypie.git@fc6467c1bdbae634dc58f885b6ef543d0b09174c#egg=django_tastypie-wlanslovenija
---allow-all-external git+https://github.com/wlanslovenija/django-datastream.git@b32b02cfaee0a3b722930bd3b401598020b72ed8#egg=django_datastream-master
-git+https://github.com/wlanslovenija/django-guardian.git@38460f0ab9c0d46e72b92a34a49d039d40a75a5e#egg=django_guardian-wlanslovenija
-git+https://github.com/wlanslovenija/django-missing.git@a068cd6f66a0b33012259ebdc335b5082b772093#egg=django_missing-master
-git+https://github.com/wlanslovenija/django-overextends.git@d70cdf44bff3fd130a0a64f4f6836ca3b8ea030f#egg=django_overextends-wlanslovenija
+--allow-all-external git+https://github.com/wlanslovenija/django-datastream.git@fe61aace1b8c8131ae914b067f9f2a200fedbeec#egg=django_datastream-master
+django-guardian==1.3
+git+https://github.com/wlanslovenija/django-missing.git@028bb5a94e82a0f290cbd6fb85edf7163fcdf05c#egg=django_missing-master
+git+https://github.com/wlanslovenija/django-overextends.git@83d29ef10c781bb751e0cf6f55411c9f44a848dd#egg=django_overextends-wlanslovenija
 django-phonenumber-field==0.5
 django_polymorphic==0.7.1
-git+https://github.com/wlanslovenija/django-registration.git@670779dce3b81fabc7cc2394db1ddd45ffcc4dc7#egg=django_registration-wlanslovenija
-django-sekizai==0.8.1
+git+https://github.com/ubernostrum/django-registration.git@6a953b0044a7e82fca0f9ce3b7146a0dffec3a51#egg=django_registration-master
```

Comment:
Ohhh. I think I have changed `django-registration` quite a bit from the original package. Not sure if just switching a package can really work. Please verify this. I assume we will have to update the whole registration code anyway. To custom Django user model, registration, login and logout notifications based on official signals, and even OAuth support (both as a consumer and service).